tools/toolchain/scripts/install_openblas.sh

```diff
@@ -120,6 +122,7 @@ case "$with_openblas" in
         echo "==================== Finding LAPACK from system paths ===================="
         # assume that system openblas is threaded
         check_lib -lopenblas "OpenBLAS"
+        check_lib -lopenblas_omp 2>/dev/null && OPENBLAS_LIBS_OMP="-lopenblas_omp"
```

Comment:
while at it you may want to check for `-lopenblas_openmp` as well. On an openSUSE system:

```console
$ ls -1 /usr/lib64/libopenblas*
/usr/lib64/libopenblas_openmp.so
/usr/lib64/libopenblas_openmp.so.0
/usr/lib64/libopenblas_pthreads.so
/usr/lib64/libopenblas_pthreads.so.0
/usr/lib64/libopenblas.so
/usr/lib64/libopenblas.so.0
```

Reply:
Sounds good. I guess it mainly makes sense here and not in the "user paths" section?

Reply:
I don't think we need this in the "user path" section, because `-lopenblas_omp` is the default after all.

Reply:
Not if this change is applied. If `OPENBLAS_LIBS_OMP` is not set, `-lopenblas` would be used:
https://github.com/cp2k/cp2k/pull/937/files#diff-27faa3da481dac7fc5280ec3a0117d58R144

Reply:
Maybe we should just wait until I've merged #932 (hopefully sometime this week)?

Reply:
Ah, I did not notice that #932 affects this. Waiting for that makes sense. If I see correctly, that will make this change even more straightforward as we only need to deal with one linker flag.

Reply:
I've merged #932. As expected, this PR now has some conflicts.